yt_dlp/extractor/playsuisse.py

```diff
@@ -5,10 +5,11 @@
 
 
 class PlaySuisseIE(InfoExtractor):
-    _VALID_URL = r'https?://(?:www\.)?playsuisse\.ch/watch/(?P<id>[0-9]+)'
+    _VALID_URL = r'https?://(?:www\.)?playsuisse\.ch/(watch|detail)/(.*\?episodeId=)?(?P<id>[0-9]+)?'
     _TESTS = [
         {
-            'url': 'https://www.playsuisse.ch/watch/763211/0',
```

Comment:
Keep the old test too

Reply:
Why, this format doesn't exist and work anymore?

Reply:
The URL still opens in browser and so should remain supported

Reply:
Only this is pending